src/main/java/ome/formats/importer/targets/ModelImportTarget.java

```diff
@@ -110,17 +121,35 @@ public void init(String target) {
     }
 
     public Class<? extends IObject> getObjectType() {
+        if (subTarget != null) {
+            return subTarget.getObjectType();
+        }
         return type;
     }
 
     public Long getObjectId() {
+        if (subTarget != null) {
+            return subTarget.getObjectId();
+        }
         return id;
     }
 
     @Override
     public IObject load(OMEROMetadataStoreClient client, ImportContainer ic) throws Exception {
         IQueryPrx query = client.getServiceFactory().getQueryService();
         IUpdatePrx update = client.getServiceFactory().getUpdateService();
+
+        if ("Project".equals(simpleName)) {
+            if (!name.contains("/")) {
+                throw new RuntimeException("Project name is missing a '/': " + name);
```

Comment:
Perhaps complicating things, nightshade has a fair few projects whose name contains a `/`. Might want to pick a different separator. Examples:

- Mad2/Mad1
- Analysis Sgo2 PP2A-A a/b
- 366-Hec/MCAK
- stuck in late prometaphase/metaphase
- Y1-114 MDC1-GFP +/- Bod1L siRNA

Reply:
Suggestions for one that will work in general? The other option is to introduce escaping, but it quickly gets gnarly. (Another option I considered was to only create a project if the dataset isn't _already_ in a project, but again....tricky.)

Reply:
I'm trying to figure if it makes sense to search for where we already have a target where `p.name || d.name = :foo || '/' || bar` or perhaps use `:foo LIKE (p.name || '%')` to find projects with a matching prefix then figuring from that which `/` to act on. Agreed that it'd be lovely to avoid escaping.

Reply:
If I give up on trying to make it class agnostic (which it already isn't), I imagine we could also just split on `/Dataset:`.

Reply:
Looked briefly at searching for `p.name || d.name` but that assumes too much knowledge of the Dataset subTarget for the current implementation. e.g. what happens if someone passes an id rather than a name? I've pushed a commit to use `/Dataset:` as the separator for comments.